pkg/kubernetes/namespace/v1alpha1/namespace.go

```diff
@@ -49,6 +49,12 @@ func (b *Builder) WithName(name string) *Builder {
 	return b
 }
 
+// WithGeneratedName appends a random string after the name
+func (b *Builder) WithGeneratedName(name string) *Builder {
```

Comment:
Can this be renamed to `WithGenerateName`
No need to use past tense

Reply:
Done